pkg/internal/myownsanity/number_test.go

```diff
@@ -0,0 +1,12 @@
+package myownsanity
+
+import (
+	"github.com/stretchr/testify/assert"
+	"testing"
+)
+
+func TestMax(t *testing.T) {
+	assert.Equal(t, 2, Max(1, 2))
+	assert.Equal(t, 1000, Max(1000, 100))
+	assert.Equal(t, 500, Max(500, 500))
```

Comment:
This is such a stupid test, but if it's going to be here then tests for the other silly methods should be added as well.

Higher code coverage = happy chemical.